hugo/layouts/papers/list-entry.html

```diff
@@ -14,9 +14,11 @@
     </a>
     {{- end -}}
   {{- else -}}
+    {{- if isset $paper "url" -}}
```

Comment:
Should be `with $paper.url` and a dot in place of `$paper.url` in the next line

Reply:
Would your proposed change handle if there were no `url` member?

Reply:
Yes.